J/jlqml/build_tarballs.jl

```diff
@@ -54,9 +54,9 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("libcxxwrap_julia_jll"),
-    Dependency("Qt6Declarative_jll"),
+    Dependency("Qt6Declarative_jll"; compat="6.4.2"),
```

Comment:
Still missing `=` here?

Reply:
No, for packages not part of Qt itself we can be less strict